address/book.go

```diff
@@ -193,15 +194,70 @@ func NewBook(cfg BookConfig) *Book {
 	}
 }
 
+// queryAssetInfo attempts to locate asset genesis information by querying
+// geneses already known to this node. If asset issuance was not previously
+// verified, we then query universes in our federation for issuance proofs.
+func (b *Book) queryAssetInfo(ctx context.Context,
+	id asset.ID) (*asset.AssetGroup, error) {
+
+	// Check if we know of this asset ID already.
+	assetGroup, err := b.cfg.Store.QueryAssetGroup(ctx, id)
+	switch {
+	case assetGroup != nil:
+		return assetGroup, nil
+
+	// Asset lookup failed gracefully; continue to asset lookup using the
+	// AssetSyncer if enabled.
+	case errors.Is(err, ErrAssetGroupUnknown):
+		if !b.cfg.SyncerActive {
```

Comment:
Why would this ever be false in practice?

Reply:
Intended as user config option; setting syncer to `nil` on startup would also work.

Reply:
Gotcha, I think `nil` here matches how we manage optional interface impls elsewhere.

Reply:
And that works without this being `*AssetSyncer`?